superset/assets/src/SqlLab/components/ResultSet.jsx

```diff
@@ -51,7 +51,7 @@ const defaultProps = {
   database: {},
 };
 
-const SEARCH_HEIGHT = 46;
+const SEARCH_HEIGHT = 186;
```

Comment:
hmm, I think the hardcode here is the height of the `ResultSetControl` div. It appears to be 46 pixels in both Chrome and FF. Are we sure about 186 here?

Reply:
as it was, the result table was going out of the screen (last row never visible, unless you would scroll the outer panel)

I'm generally against magic numbers in the code though, so we should probably find a proper solution for it (ie calculating the sibling heights after rendering) :)